code/modules/mapfluff/ruins/lavalandruin_code/elephantgraveyard.dm

```diff
@@ -271,6 +276,29 @@
 		deconstruct(TRUE)
 		return TRUE
 
+/obj/structure/closet/crate/grave/container_resist_act(mob/living/user)
+	if(opened)
+		return
+	// The player is trying to dig themselves out of an early grave
+	user.changeNext_move(CLICK_CD_BREAKOUT)
+	user.last_special = world.time + CLICK_CD_BREAKOUT
+	user.visible_message(
+		span_warning("[src]'s dirt begins to shift and rumble!"),
+		span_notice("You desperately begin to claw at the dirt around you, trying to force yourself upwards through the soil... (this will take about [DisplayTimeText(breakout_time)].)"), \
+		span_hear("You hear the sound of shifting dirt from [src].")
+		)
+	if(do_after(user, breakout_time, target = src))
+		if(opened)
+			return
+		user.visible_message(
+			span_danger("[user] emerges from [src], scattering dirt everywhere!"),
+			span_notice("You triumphantly surface out of [src], scattering dirt all around the grave!")
```

Comment:
```suggestion
			span_notice("You triumphantly surface out of [src], scattering dirt all around the grave!"),
```

trailing commas good

Reply:
Not that I disagree with you, I agree, but I am curious as to the theory as to why if you wouldnt mind

Reply:
it's the codebase standard because in case someone wants to extend it (like adding the hearable message) they don't have to add the trailing comma on code they didn't write. it also makes the github diff look less ugly

Reply:
interesting, thanks